### PR TITLE
Add support for year, yearmonth and duration field types

### PR DESCRIPTION
**Pull request by rflprr**

### CODE REVIEW

File: tests/test_types.py [hunk 1]
```diff
@@ -204,38 +205,6 @@ def test_number_type_with_already_cast_value(self):
                 self.assertEqual(_type.cast(value), value)
 
 
-class TestInteger(base.BaseTestCase):
```
Comment (roll) [inline tests/test_types.py]: It seems `IntegerType` test has been removed by mistake.

**Comment by roll** [comment]
Also please take a look on lint output leading to build failure - https://travis-ci.org/frictionlessdata/jsontableschema-py/jobs/214419196

**Comment by rflprr** [comment]
@roll Looks like Travis is happy now, but the coverage check failed. Any suggestions?

**Comment by roll** [comment]
@rflprr 
No worries about it. Merging. Thanks!

**Comment by roll** [comment]
@rflprr 
It's released - https://pypi.python.org/pypi/jsontableschema/0.10.0

**Comment by rflprr** [comment]
Great! Thanks for the heads-up. 
Any thoughts on this one: https://github.com/frictionlessdata/jsontableschema-pandas-py/pull/23 ?

Separately, I noticed that `yearmonth` is described in the spec as a 1-2 digit field. However, the `gYearMonth` type referenced in the spec uses a `MM-YYYY` format. Either one is fine, in my view, but the spec should be made more clear one way or another.

**Comment by roll** [comment]
@rflprr 
I'm almost on pandas PR) Will review pretty soon.

About specs - I'm doing a high-level analysis of the whole spec - https://github.com/frictionlessdata/jsontableschema-py/issues/150 - so I'll send a issue to `specs` tracker based on your words.

**Pull request merged**